windev.ps1

```diff
@@ -1,55 +1,171 @@
 <#
 .SYNOPSIS
-    This Script is used as a target for the https://christitus.com/windev alias.
-    It queries the latest winget release (no matter if Pre-Release, Draft or Full Release) and invokes It
+    This script is used as a target for the https://christitus.com/windev alias.
+    It queries the latest WinUtil release (no matter if it's a Pre-Release, Draft, or Full Release) and runs it.
 .DESCRIPTION
-    This Script provides a simple way to always start the bleeding edge release even if it's not yet a full release.
-    This function should be run with administrative privileges.
-    Because this way of recursively invoking scripts via Invoke-Expression it might very well happen that AV Programs flag this because it's a common way of mulitstage exploits to run
+    This script provides a simple way to start the bleeding edge release even if it's not yet a full release.
+    This function can be run both with administrator and non-administrator permissions.
+    If it is not running as administrator, it will attempt to relaunch itself with administrator permissions.
+    The script no longer uses Invoke-Expression for its execution and now relies on Start-Process.
 .EXAMPLE
-    irm https://christitus.com/windev | iex
+    Run in PowerShell > irm https://christitus.com/windev | iex
     OR
-    Run in Admin Powershell >  ./windev.ps1
+    Run in PowerShell > .\windev.ps1
+    OR
+    Run in PowerShell > iex "& { $(irm https://christitus.com/windev) } <arguments>"
+    OR
+    Run in PowerShell > .\windev.ps1 <arguments>
+.NOTES
```

Comment:
Is there a need for .NOTES and .EXAMPLE to be the same?

Reply:
Oh, I didn't even catch that when I was writing it! It's been removed now.